src/main/resources/data/households_flanders.json

```diff
@@ -0,0 +1,8768 @@
+[
```

Comment:
Also, this is a great way to inflate your stats for this project. Why didn't I think of that? :stuck_out_tongue_winking_eye: